test/test_util.jl

```diff
@@ -55,8 +55,8 @@ function adjoint_test(
     f, ȳ, x...; rtol=_rtol, atol=_atol, fdm=central_fdm(5, 1), print_results=false
 )
     # Compute forwards-pass and j′vp.
-    y, back = Zygote.pullback(f, x...)
-    adj_ad = back(ȳ)
+    _f = (x) -> f(x...)
+    y, adj_ad = DI.value_and_pullback(_f, DI.AutoMooncake(), x, ȳ)
```

Comment:
```suggestion
    y, adj_ad = DI.value_and_pullback(_f, DI.AutoMooncake(), x, (ȳ, ))
```
In DI, tangents and cotangents are passed as tuples to enable the batched behavior of ForwardDiff and Enzyme.